float32 x
float32 y
float32 z
---
bool success
float64 latitude
float64 longitude
float64 altitude
